Showing Previous Response Header gives crash

**Describe the bug**
Giving multiple wrong answers shows up the previous header which crashes the app. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Open any exploration
2. Try to give multiple wrong answers -> crash

**Expected behaviour**
It should not crash and show the previous header properly

**Device**
Pixel XL
Redmi Note 5 Pro

**Error**
```
android.view.InflateException: Binary XML file line #85: Binary XML file line #85: You must supply a layout_height attribute.
Caused by: java.lang.UnsupportedOperationException: Binary XML file line #85: You must supply a layout_height attribute.
```

--- a/app/src/main/res/layout-land/previous_responses_header_item.xml
+++ b/app/src/main/res/layout-land/previous_responses_header_item.xml
@@ -23,7 +23,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="1dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toStartOf="@+id/previous_responses_header_text"
@@ -48,7 +48,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="1dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toEndOf="parent"

--- a/app/src/main/res/layout-sw600dp-land/previous_responses_header_item.xml
+++ b/app/src/main/res/layout-sw600dp-land/previous_responses_header_item.xml
@@ -59,7 +59,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="2dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toStartOf="@+id/previous_responses_header_text"
@@ -84,7 +84,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="2dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toEndOf="parent"

--- a/app/src/main/res/layout-sw600dp-port/previous_responses_header_item.xml
+++ b/app/src/main/res/layout-sw600dp-port/previous_responses_header_item.xml
@@ -59,7 +59,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="2dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toStartOf="@+id/previous_responses_header_text"
@@ -84,7 +84,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="2dp"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toEndOf="parent"

--- a/app/src/main/res/layout/previous_responses_header_item.xml
+++ b/app/src/main/res/layout/previous_responses_header_item.xml
@@ -59,7 +59,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1}"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toStartOf="@+id/previous_responses_header_text"
@@ -84,7 +84,7 @@
 
     <FrameLayout
       android:layout_width="0dp"
-      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1}"
+      android:layout_height="@{viewModel.isSplitView? @dimen/thickness_2 : @dimen/thickness_1, default=@dimen/thickness_1}"
       android:background="@color/mid_grey"
       app:layout_constraintBottom_toBottomOf="@+id/previous_responses_header_text"
       app:layout_constraintEnd_toEndOf="parent"

